.vscode/settings.json

```diff
@@ -15,4 +15,12 @@
     "[javascriptreact]": {
         "editor.defaultFormatter": "esbenp.prettier-vscode"
     },
+    "[java]": {
```

Comment:
Not working when I test locally. Did you test this?

Reply:
I have tested it, it worked locally. Maybe something in the configuration is messed up? I'll check this out tmr

Reply:
after some further digging it looks like the activation events for this plugin are
workspaceContains:gradlew
workspaceContains:gradlew.bat

and since gradle is inside of the children folders and not the parent workspace i dont think it will work. Ill make a new repo and test this out

Reply:
Yup that was the issue. It works as intended in the new repository, even lints when something is not spotless

<img width="592" alt="Screenshot 2024-10-25 at 1 03 35 AM" src="https://github.com/user-attachments/assets/52ae88ac-c848-407e-a5ef-c8af118079d0">

Reply:
There is another extension which is more bare bones thats called Run On Save. the only issue im facing currently is how to determine where the java project was created at.

Reply:
Nice. I was proud to see your feature request in the plugin repo when I walked the TAs through the issue.

Reply:
Thank you!